terraform-aws-github-runner/modules/runners/lambdas/runners/src/scale-runners/runners.ts

```diff
@@ -514,142 +514,58 @@ export async function tryReuseRunner(
   const ssmM: Map<string, SSM> = new Map();
 
   for (const runner of runners) {
-    if (runner.ghRunnerId === undefined) {
-      console.debug(`[tryReuseRunner]: Runner ${runner.instanceId} does not have a GithubRunnerID tag`);
-      continue;
-    }
-    if (runner.awsRegion === undefined) {
-      console.debug(`[tryReuseRunner]: Runner ${runner.instanceId} does not have a region`);
-      continue;
-    }
-    if (runner.org === undefined && runner.repo === undefined) {
-      console.debug(`[tryReuseRunner]: Runner ${runner.instanceId} does not have org or repo`);
-      continue;
-    }
-
-    if (runner.stage === EphemeralRunnerStage.RunnerReplaceEBSVolume) {
-      console.debug(
-        `[tryReuseRunner]: Runner ${runner.instanceId} the runner is in RunnerReplaceEBSVolume stage, skip to reuse it`,
-      );
+    // check if runner is reusable
```

Comment:
Imo this comment is not necessary and actually contradicts the code. It says "check if runner is reusable" but the code is checking if the runner is NOT reusable.

I'd recommend just removing the comment altogether.